l3backend/l3backend-opacity.dtx

```diff
@@ -207,26 +195,26 @@
 % \end{macro}
 %
 % \begin{macro}{\@@_backend_fill:n, \@@_backend_stroke:n}
-% \begin{macro}{\@@_backend_fill_stroke:nn, \@@_backend_fill_stroke:ee}
+% \begin{macro}{\@@_backend_fill_stroke:nn}
 %   For separate fill and stroke, we need to work out if we need to do
 %   more work or if we can stick to a single setting.
 %    \begin{macrocode}
 \cs_new_protected:Npn \@@_backend_fill:n #1
   {
-    \@@_backend_fill_stroke:ee
-      { \fp_eval:n { min(max(0,#1),1) } }
-      \l_@@_backend_stroke_tl
+    \exp_args:Nno \@@_backend_fill_stroke:nn
```

Comment:
V-type

Reply:
Is this for clean coding or necessary? "o" is always faster than "v" so I would lean towards "o" in core code.